src/components/EditTiers.js

```diff
@@ -104,6 +104,10 @@ class EditTiers extends React.Component {
         id: 'tier.goal.label',
         defaultMessage: 'Goal',
       },
+      'button.label': {
+        id: 'tier.button.label',
+        defaultMessage: 'Button text',
```

Comment:
When adding a new message, you should run `npm run build:langs` and commit the result.

Reply:
Also unable to run this on windows, this was the error message I got:

```
rm' is not recognized as an internal or external command,
operable program or batch file
```

Reply:
Unfortunately, we don't have windows users in our small team, this is why some things are broken. We really want to fix this.

In this case, I'm gonna replace `rm` with `rimraf`, but there is certainly more.

Reply:
ok, what do you suggest I do.

Reply:
This is not an important issue, this can be taken care of by us if necessary.

Maybe let me know if this branch https://github.com/opencollective/opencollective-frontend/tree/rimraf fix the issue.